src/ucs/config/parser.c

```diff
@@ -1855,7 +1855,7 @@ void ucs_config_parser_print_all_opts(FILE *stream, const char *prefix,
 static void ucs_config_parser_concat_fuzzy_matches_recurs(
         const ucs_config_field_t *fields, const char *env_prefix,
         const char *table_prefix, const char *user_var,
-        ucs_string_buffer_t *unused_var_buff)
+        ucs_string_buffer_t *fuzzy_buff)
```

Comment:
1. `buff` -> `buffer`
2. Is the **buffer** fuzzy?

Reply:
I want to propose to replace "fuzzy" word to "similar" one. IMHO it has closer sense.